src/components/empty_state/empty_state.rs

```diff
@@ -56,7 +56,7 @@ pub fn empty_state(props: &EmptyStateProperties) -> Html {
             class={classes!(
                 EmptyStateStyles::EMPTY_STATE,
                 props.variant.classes(props.full_height).to_string(),
-                &props.class,
+                props.class.to_string(),
```

Comment:
I think this can just be `props.class.clone()`. Otherwise it would need to be re-parsed during runtime.

Reply:
Some for the other occurrences.

Reply:
Thanks, I've just made that change. Thanks for pointing this out. I didn't know `.to_string` and `.clone` were different, but I know